### PR TITLE
Feat: Create Development section - iOS development page

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [x] Laptop / Macbook (chrome, firefox, safari)
- [ ] iPhone(safari)
- [x] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

**Description:**
Create Development section for ios page
**Mobile Performance:**
image

**Desktop Image:**
image

**Video:**
https://github.com/canopas/canopas-website/assets/98794636/308a17ed-6266-4f2d-aa4c-e767f56d3c16

**Performance on mobile:**
Screenshot from 2023-07-06 16-16-08

**Performance on desktop:**
Screenshot from 2023-07-20 16-23-14

### CODE REVIEW

File: vue-frontend/src/components/ios-app-development/DevelopmentSection.vue [hunk 1]
```diff
@@ -0,0 +1,260 @@
+<template>
+  <div class="tw-my-16 tw-flex tw-flex-col xll:tw-container xll:tw-my-[4rem]">
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/ios-app-development/DevelopmentSection.vue:2]: Use default class

File: vue-frontend/src/components/ios-app-development/DevelopmentSection.vue [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+    <div class="tw-container tw-mb-2.5 tw-flex tw-flex-col tw-text-center">
+      <span
+        class="tw-tracking-[-1.2px] tw-mb-2.5 tw-font-inter-bold tw-text-[1.875rem] tw-leading-[2.4375rem] tw-text-black-core/[0.87] md:!tw-mx-0 md:!tw-w-[100%] md:tw-text-[2.65625rem] lg:tw-text-[3.4375rem] md:tw-leading-[3.796875rem] lg:tw-leading-[5.15625rem] xs:tw-mx-auto xs:tw-w-[71%]"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/ios-app-development/DevelopmentSection.vue:5]: same applied here and check throughout the code

File: vue-frontend/src/components/ios-app-development/DevelopmentSection.vue [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+    <div
+      class="tw-hidden md:tw-flex tw-flex-row tw-container tw-mt-16 tw-bg-[#FFF] tw-drop-shadow-[0px_0px_20px_rgba(0,0,0,0.10)]"
+    >
+      <div class="tw--ml-4">
+        <div
+          v-for="(item, index) in items"
+          :key="index"
+          :class="activeIndex == index ? 'tw-border-r-[#ff835b]' : ''"
+          class="tw-flex tw-flex-col tw-justify-center tw-items-center tw-h-[7.5rem] lg:tw-h-36 tw-w-36 lg:tw-w-60 tw-bg-[#F5F5F5] tw-border-b tw-border-[#E3E3E3] tw-border-r-[5px]"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/ios-app-development/DevelopmentSection.vue:84]: Box color is also changing here please refer figma

File: vue-frontend/src/components/ios-app-development/DevelopmentSection.vue [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+        class="tw-tracking-[-1.2px] tw-mb-2.5 tw-font-inter-bold tw-text-[1.875rem] tw-leading-[2.4375rem] tw-text-black-core/[0.87] md:!tw-mx-0 md:!tw-w-[100%] md:tw-text-[2.65625rem] lg:tw-text-[3.4375rem] md:tw-leading-[3.796875rem] lg:tw-leading-[5.15625rem] xs:tw-mx-auto xs:tw-w-[71%]"
+      >
+        iOS App Development Services</span
+      >
+      <span
+        class="tw-font-inter-regular tw-text-[1rem] md:tw-text-[1.25rem] tw-leading-[1.5rem] md:tw-leading-[1.875rem] tw-text-black-core/[0.60] md:tw-font-inter-medium lg:tw-text-[1.5rem] lg:tw-leading-[2.25rem] lg:tw-px-20 2xl:tw-px-56"
+        >With a blend of robust technology, intuitive designs, and innovative
+        strategies, our team can help you craft high-quality, standout
+        user-friendly apps that drive business growth and customer
+        engagement.</span
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/ios-app-development/DevelopmentSection.vue:14]: This paragraph is not properly aligned according to figma

**Review by cp-mansi-d — DISMISSED** [review]

**Review by cp-jagruti-a — APPROVED** [review]

**Review by cp-mansi-d — APPROVED** [review]

**Pull request merged**